### ISSUE TITLE
I'm having trouble using accents.

### ISSUE THREAD

**Issue by jcmljunior** [flathub/org.blender.Blender#143]
Hello. I am using the "Fedora Silver Blue" system and this problem only happens with Blender.
Accents like: "áàâ" do not work."

**Comment by kevinsmia1939** [comment]
What does not work specifically? Naming the objects with áàâ?

**Comment by jcmljunior** [comment]
> What does not work specifically? Naming the objects with áàâ?

Yes

**Comment by jcmljunior** [comment]
https://github.com/flathub/org.blender.Blender/assets/82642883/77259021-77cc-4e5d-9120-d907f5d83615

**Comment by kevinsmia1939** [comment]
Have you test this with non-flatpak Blender? Like from the blender.org?

**Comment by jcmljunior** [comment]
> Have you test this with non-flatpak Blender? Like from the blender.org?

Yes. It worked normally. The problem only happens in the version with flatpak.

**Comment by kevinsmia1939** [comment]
Does copy and pasting the character rather than typing work?

**Comment by jcmljunior** [comment]
> Does copy and pasting the character rather than typing work?

I will check this out as soon as possible.

**Comment by jcmljunior** [comment]
Copy and Paste working for me.
![Captura de tela de 2023-07-05 06-32-25](https://github.com/flathub/org.blender.Blender/assets/82642883/7de5143f-8dbb-4dac-82e0-79b0ff3bde1c)

**Comment by kevinsmia1939** [comment]
Hmm, I honestly have no clue, there seems to be an issue with adding accent when you type.

**Comment by jcmljunior** [comment]
After I make the key combinations, the following error appears on the command line.
flatpak run org.blender.Blender
```
Read prefs: "/var/home/julio/.var/app/org.blender.Blender/config/blender/3.6/config/userpref.blend"
Failed to open dir (No such file or directory): /run/user/1000/gvfs/
register
Bad keycode lookup. Keysym 0x0 Status: XLookupNone
'' 0x7f92087dca80 0x7f9208bfe100
Bad keycode lookup. Keysym 0x0 Status: XLookupNone
'' 0x7f92087dca80 0x7f9208bfe100
Bad keycode lookup. Keysym 0x0 Status: XLookupNone
'' 0x7f92087dca80 0x7f9208bfe100
Bad keycode lookup. Keysym 0x0 Status: XLookupNone
'' 0x7f92087dca80 0x7f9208bfe100
Bad keycode lookup. Keysym 0x0 Status: XLookupNone
'' 0x7f92087dca80 0x7f9208bfe100
Bad keycode lookup. Keysym 0x0 Status: XLookupNone
'' 0x7f92087dca80 0x7f9208bfe100
Bad keycode lookup. Keysym 0x0 Status: XLookupNone
'' 0x7f92087dca80 0x7f9208bfe100
Bad keycode lookup. Keysym 0x0 Status: XLookupNone
'' 0x7f92087dca80 0x7f9208bfe100
Bad keycode lookup. Keysym 0x0 Status: XLookupNone
'' 0x7f92087dca80 0x7f9208bfe100
Saved session recovery to "/tmp/quit.blend"
unregister

Blender quit
```

**Comment by kevinsmia1939** [comment]
Just to be sure, this key combination does not crash non-flatpak blender?

**Comment by jcmljunior** [comment]
https://github.com/flathub/org.blender.Blender/assets/82642883/719bfdea-f021-4638-a77b-b81683daf9c1

> Just to be sure, this key combination does not crash non-flatpak blender?

Keeps working.

**Comment by jcmljunior** [comment]
> After I make the key combinations, the following error appears on the command line. flatpak run org.blender.Blender
> 
> ```
> Read prefs: "/var/home/julio/.var/app/org.blender.Blender/config/blender/3.6/config/userpref.blend"
> Failed to open dir (No such file or directory): /run/user/1000/gvfs/
> register
> Bad keycode lookup. Keysym 0x0 Status: XLookupNone
> '' 0x7f92087dca80 0x7f9208bfe100
> Bad keycode lookup. Keysym 0x0 Status: XLookupNone
> '' 0x7f92087dca80 0x7f9208bfe100
> Bad keycode lookup. Keysym 0x0 Status: XLookupNone
> '' 0x7f92087dca80 0x7f9208bfe100
> Bad keycode lookup. Keysym 0x0 Status: XLookupNone
> '' 0x7f92087dca80 0x7f9208bfe100
> Bad keycode lookup. Keysym 0x0 Status: XLookupNone
> '' 0x7f92087dca80 0x7f9208bfe100
> Bad keycode lookup. Keysym 0x0 Status: XLookupNone
> '' 0x7f92087dca80 0x7f9208bfe100
> Bad keycode lookup. Keysym 0x0 Status: XLookupNone
> '' 0x7f92087dca80 0x7f9208bfe100
> Bad keycode lookup. Keysym 0x0 Status: XLookupNone
> '' 0x7f92087dca80 0x7f9208bfe100
> Bad keycode lookup. Keysym 0x0 Status: XLookupNone
> '' 0x7f92087dca80 0x7f9208bfe100
> Saved session recovery to "/tmp/quit.blend"
> unregister
> 
> Blender quit
> ```

The same issue occurs in other distributions, such as Pop-OS, for example.
